app/src/features/projects/view/ProjectPage.test.tsx

```diff
@@ -111,6 +137,184 @@ describe('ProjectPage', () => {
+    mockBiohubApi().project.deleteProject.mockResolvedValue(true);
+
+    const authState = {
+      keycloakWrapper: {
+        keycloak: {
+          authenticated: true
+        },
+        hasLoadedAllUserInfo: true,
+        systemRoles: [SYSTEM_ROLE.PROJECT_ADMIN] as string[],
+        getUserIdentifier: () => 'testuser',
+        hasAccessRequest: false,
+        hasSystemRole: () => true,
+        getIdentitySource: () => 'idir',
+        username: 'testusername',
+        displayName: 'testdisplayname',
+        email: 'test@email.com',
+        firstName: 'testfirst',
+        lastName: 'testlast',
+        refresh: () => {}
+      }
+    };
+
+    const { getByTestId, findByText } = render(
+      <AuthStateContext.Provider value={(authState as unknown) as IAuthState}>
+        <DialogContextProvider>
+          <Router history={history}>
+            <ProjectPage />
+          </Router>
+        </DialogContextProvider>
+      </AuthStateContext.Provider>
+    );
+
+    const projectHeaderText = await findByText('Test Project Name', { selector: 'h1' });
+    expect(projectHeaderText).toBeVisible();
+
+    expect(getByTestId('delete-project-button')).toBeDisabled();
+  });
+
+  it('sees delete project button as enabled when accessing an unpublished project as a project administrator', async () => {
+    mockBiohubApi().codes.getAllCodeSets.mockResolvedValue({
+      activity: [{ id: 1, name: 'activity 1' }]
+    } as any);
+    mockBiohubApi().project.getProjectForView.mockResolvedValue({
+      ...getProjectForViewResponse,
+      project: { ...getProjectForViewResponse.project, publish_date: '' }
+    });
+    mockBiohubApi().project.deleteProject.mockResolvedValue(true);
+
+    const authState = {
+      keycloakWrapper: {
+        keycloak: {
+          authenticated: true
+        },
+        hasLoadedAllUserInfo: true,
+        systemRoles: [SYSTEM_ROLE.PROJECT_ADMIN] as string[],
+        getUserIdentifier: () => 'testuser',
+        hasAccessRequest: false,
+        hasSystemRole: () => true,
+        getIdentitySource: () => 'idir',
+        username: 'testusername',
+        displayName: 'testdisplayname',
+        email: 'test@email.com',
+        firstName: 'testfirst',
+        lastName: 'testlast',
+        refresh: () => {}
+      }
+    };
+
+    const { getByTestId, findByText } = render(
+      <AuthStateContext.Provider value={(authState as unknown) as IAuthState}>
+        <DialogContextProvider>
+          <Router history={history}>
+            <ProjectPage />
+          </Router>
+        </DialogContextProvider>
+      </AuthStateContext.Provider>
+    );
+
+    const projectHeaderText = await findByText('Test Project Name', { selector: 'h1' });
+    expect(projectHeaderText).toBeVisible();
+
+    expect(getByTestId('delete-project-button')).toBeEnabled();
+  });
+
+  it('sees delete project button as disabled when accessing a published project as a project administrator', async () => {
```

Comment:
question for @jeznorth 
do we want button disabled or do we want it to be enabled but then an error to show up when clicked